### PR TITLE
feat: Improving docs by adding start information

### PR DESCRIPTION
**Pull request by SanthoshSiddegowda**
Adding information related to starting the application.

### CODE REVIEW

**Comment by WendellAdriel** [comment]
@SanthoshSiddegowda sorry for the delay on this, can you please fix the conflicts?

File: README.md [hunk 1]
```diff
@@ -83,6 +83,20 @@ Update the admin user in the `database/seeders/DatabaseSeeder.php` file and run
...
 
+## Running the Application
+
+Run this command to start the app
+
+```bash
+make start
+```
+
+After completion, verify by visiting the endpoint below
```
Comment (WendellAdriel) [inline README.md:94]: Replace endpoint for URL

File: README.md [hunk 2]
```diff
@@ -83,6 +83,20 @@ Update the admin user in the `database/seeders/DatabaseSeeder.php` file and run
...
+
+```bash
+http://localhost:8000/
```
Comment (WendellAdriel) [inline README.md:97]: Here the port can be anything set in the APP_EXTERNAL_PORT env variable, make sure to update this to include this info.

**Review by WendellAdriel — CHANGES_REQUESTED** [review]
Fix the conflicts in the file and work on the comments that were added, please.

File: README.md (hunk 2 above)
Comment (SanthoshSiddegowda) [reply README.md:97]: working on it.

File: README.md (hunk 1 above)
Comment (SanthoshSiddegowda) [reply README.md:94]: working on it

**Comment by WendellAdriel** [comment]
This was added on #5

**Pull request closed without merging**